src/appMain/hmi_capabilities.json

```diff
@@ -242,6 +242,13 @@
                         "resolutionWidth": 35,
                         "resolutionHeight": 35
                     }
+                },
+                {
+                    "name": "alertIcon",
+                    "imageResolution": {
+                        "resolutionWidth": 225,
```

Comment:
Part of me says to change this to 35 for consistency, but it doesn't matter too much